tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -489,31 +489,38 @@ stages:
               if [[ $executionTimeTestsExitCode -ne 0 ]] || [[ $memoryTestsExitCode -ne 0 ]]; then
                 exit 1;
               fi
+          env:
+            FLUID_TEST_LOGGER_PKG_PATH: ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger # Contains getTestLogger impl to inject
+            FLUID_BUILD_ID: ${{ variables.artifactBuildId }}
```

Comment:
@NicholasCouri I think this will stamp an incorrect build id in these logs, unless by design we intended something somewhat funky. `${{ variables.artifactBuildId }}` is the pipeline run id of the _client packages_ run that then triggered the perf benchmarks run, not the run id of the perf benchmarks run itself (that'd be `$(Build.BuildId)`), so the build id we see in Kusto would not point back to the correct pipelien run that generated them. I think `$(Build.BuildId)` is the correct thing to use here (and in lines 557 and 565)?

Reply:
Interesting as I was seeing the telemetry being populated :)  I've just changed it and will submit a new review.